src/openms/include/OpenMS/ANALYSIS/ID/BayesianProteinInferenceAlgorithm.h

```diff
@@ -115,7 +115,7 @@ namespace OpenMS
         std::vector<ProteinIdentification>& proteinIDs,
         std::vector<PeptideIdentification>& peptideIDs,
         bool greedy_group_resolution,
-        std::optional<const ExperimentalDesign> exp_des = std::optional<const ExperimentalDesign>());
+        const std::optional<const ExperimentalDesign>& exp_des = std::optional<const ExperimentalDesign>());
```

Comment:
this can be shortened to 
```suggestion
        const std::optional<const ExperimentalDesign> exp_des;
```
Not sure how many of these exist...